src/main/java/level7.java

```diff
@@ -0,0 +1,209 @@
+        tasks.remove(number - 1);
+        done.remove(number - 1);
+        type.remove(number - 1);
+    }
+    public static void todo(int i, String line, ArrayList<String> tasks, ArrayList<String> done, ArrayList<String> type) throws DukeException {
+        if (line.equals("todo")) {
+            throw new DukeException();
+        }
+        tasks.add(line.substring(4));
+        type.add("[T]");
+        System.out.println("Got it. I've added this task:");
+        System.out.println("  " + type.get(i) + "[ ]" + tasks.get(i));
+        System.out.println("Now you have " + (i + 1) + " tasks in the list.");
+        done.add("[ ]");
+    }
+
+    public static void deadline(int i, String line, ArrayList<String> tasks, ArrayList<String> done, ArrayList<String> type) {
+        int slash = line.indexOf("/");
+        String description = line.substring(8, slash-1);
+        String by = line.substring(slash+4);
+        tasks.add(description + " (by: " + by + ")");
+        type.add("[D]");
+        System.out.println("Got it. I've added this task:");
+        System.out.println("  " + type.get(i) + "[ ]" + tasks.get(i));
+        System.out.println("Now you have " + (i + 1) + " tasks in the list.");
+        done.add("[ ]");
+    }
+
+    public static void event(int i, String line, ArrayList<String> tasks, ArrayList<String> done, ArrayList<String> type) {
+        int slash1 = line.indexOf("/");
+        int slash2 = line.indexOf("/", slash1+1);
+        String description = line.substring(5, slash1-1);
+        String from = line.substring(slash1+6, slash2-1);
+        String to = line.substring(slash2+4);
+        tasks.add(description + " (from " + from + " to: " + to + ")");
+        type.add("[E]");
+        System.out.println("Got it. I've added this task:");
+        System.out.println("  " + type.get(i) + "[ ]" + tasks.get(i));
+        System.out.println("Now you have " + (i + 1) + " tasks in the list.");
+        done.add("[ ]");
+    }
+    public static void main(String[] args){
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        String filePath = "C:\\Users\\User\\ip\\Duke.txt";
+        String line;
+        ArrayList<String> tasks = new ArrayList<>();
+        ArrayList<String> done = new ArrayList<>();
+        ArrayList<String> type = new ArrayList<>();
+        int i = 0;
+        while (true) {
+            Scanner in = new Scanner(System.in);
+            line = in.nextLine();
+            if (line.startsWith("bye")) {
+                String toFile = "Here are the tasks in your list:";
+                for (int j = 0; j < i; j++) {
+                    try {
+                        toFile = toFile + System.lineSeparator() + (j + 1) + "." + type.get(j) + done.get(j) + tasks.get(j);
+                    } catch (IndexOutOfBoundsException e) {
+                        j++;
+                    }
```

Comment:
Exceptions should be use incase of wrong user input thus there should be an error message and not meant for like `j++`